src/components/WeeklySummariesReport/WeeklySummariesReport.jsx

```diff
@@ -35,7 +35,6 @@ export class WeeklySummariesReport extends Component {
 
   componentWillUnmount() {
     localStorage.removeItem('tabSelection');
-    window.removeEventListener("popstate");
```

Comment:
What's your reasoning behind removing the removal of this event listener? As it stands the event listener is never cleaned up before unmount, so it's leaking memory.

Reply:
Great catch! Let me double check if I accidentally removed it. Yeah I do think It should be present.